server.py

```diff
@@ -60,17 +84,28 @@ class Meta:
 
 
 class Submission(Model):
+    """
+    Submoission classdefines the submission of solution of a Question
+    Stores the information about:
+        User that submits the Solution
+        Time of Submission
+        Contest to whish The question belongs
+        Whether the soolution is correct
+    """
     user = ForeignKeyField(User)
     time = DateTimeField()
     contestProblem = ForeignKeyField(ContestProblems)
     is_correct = BooleanField()
 
     class Meta:
         database = db
+        #define the database in the main class
         indexes = ((("user", "time"), True),)
 
-
+#Establish a Connection with Database
 db.connect()
+
+"""Create Tables with specied fields in Databse"""
```

Comment:
please don't leave strings lying around in the code. Either use a comment or a docstring.